redis/gke-redis/provider.tf

```diff
@@ -0,0 +1,21 @@
+provider "google" {
+    credentials         = "${var.credentials}"
+    project             = "${var.project}"
+    region              = "${var.region}"
+    zone                = "${var.zone}"
```

Comment:
Shouldn't this be `location`?
If I recall correctly `Terraform` throws a warning to have this changed to `location` since `zone` has been/will be deprecated.